app/src/main/java/com/shagalalab/constitution/article/adapter/ArticleItemViewHolder.kt

```diff
@@ -0,0 +1,23 @@
+package com.shagalalab.constitution.article.adapter
+
+import android.text.SpannableStringBuilder
+import android.text.Spanned
+import android.text.style.StyleSpan
+import android.view.View
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.shagalalab.constitution.data.models.ArticleModel
+import kotlinx.android.synthetic.main.item_view.view.*
+
+class ArticleItemViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+    fun populateModel(model: ArticleModel) {
+        val s = SpannableStringBuilder()
+        val st = StyleSpan(android.graphics.Typeface.BOLD)
+        s.append(model.title, st, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE)
+        s.append(". ")
+        s.append(model.description)
```

Comment:
The problem with `\n` seems can be solved. It looks like the systems gets it from DB as `\\n`, try to replace it with `\n`, that should work.

Reply:
I couldn't replace \\n with \n. I have tried with String.replace and with "for" loop. I didn't understand why it is not working. And I have checked \\n in database, there is written as "\n". How it can get it as \\n?

Reply:
Have you tried replacing `\\n` with `\n`?

Reply:
yes

Reply:
this one works for me, strange if you couldn't end up with the similar solution even after I told you on what to do:
```
s.append(it.description.replace("\\n", "\n"))
```